climada/engine/test/test_impact.py

```diff
@@ -542,9 +542,9 @@ def test_risk_trans_pass(self):
         self.assertTrue(np.allclose(imp_rt.eai_exp, np.array([])))
         self.assertTrue(np.allclose(imp_rt.at_event, np.array([0, 0, 0, 1, 2, 3, 4, 5, 6, 10])))
         self.assertAlmostEqual(imp_rt.aai_agg, 6.2)
-        
+
```

Comment:
The editor I used for the search-and-replace automatically removed all trailing whitespaces in the files. I think this is a positive side-effect.